app/assets/stylesheets/components/_verify.scss

```diff
@@ -0,0 +1,6 @@
+@use '../variables/app' as *;
+
+.styled-as-h4 {
+  font-size: $h4;
+  display: inline;
```

Comment:
Also kinda surprised that you need to inline this?

Reply:
I was too - but the default h4 uses `display: flex` and caused a large space to appear between the headings.

Flex:

<img width="527" alt="display_flex" src="https://github.com/18F/identity-idp/assets/35475380/f4a6ac53-cf6e-4834-8d5c-a96b69846f15">

Inline:

<img width="522" alt="display_inline" src="https://github.com/18F/identity-idp/assets/35475380/8c2ba6c8-936a-4fda-91e2-d1a3cf11986f">

I noticed before the `h2` change the display was set to inline, so I went with that.

Happy to consider a different solution though.

Reply:
I think the space is due to the default margins of a heading. If we want this to appear visually as an `h4` with no margins, I might suggest:

```html
<h2 class="h4 margin-y-0">...</h2>
```

Reply:
Thank you @aduth and @JackRyan1989 ! I've tested both of your solutions locally - `.margin-y-0` and `.display-inline` both appear to work. I tested on different screen sizes as well and don't notice any issues.

Is there any reason to choose controlling this via the margin over the display attribute, or vice versa?

CSS is not my forte and I'd love to learn if there's a benefit in one over the other.

Reply:
In my opinion, if the goal is to remove the spacing, it'd be more explicit to override the margins. Changing to inline display may work because the margins (which are still present) have no effect, but it's more indirect and could be difficult for a future maintainer to know that the spacing was the primary goal of the styling.

Reply:
Thank you for clarifying that! I've pushed up that change.